common/exception/TypeDBClientException.java

```diff
@@ -98,6 +109,25 @@ private static boolean isUnimplementedMethod(StatusRuntimeException statusRuntim
         return statusRuntimeException.getStatus().getCode() == Status.Code.UNIMPLEMENTED;
     }
 
+    private static TypeDBClientException tryGetRstStreamCause(StatusRuntimeException sre) {
+        if (sre.getStatus().getCode() == Status.Code.UNAVAILABLE) {
+            if (sre.getCause() instanceof javax.net.ssl.SSLHandshakeException) {
+                if (sre.getCause().getMessage().contains("PKIX path building failed")) {
```

Comment:
oof okay is there no better way to do this rather than checking the strings :'( What if they change their messages

Reply:
Sure, let's go with that